### PR TITLE
Fix doCheck

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -17,7 +17,7 @@ python3.pkgs.buildPythonApplication rec {
   ];
 
   # does not work in sandbox
-  dontCheck = false;
+  doCheck = false;
```
Comment (Mic92) [inline default.nix:20]: Looks like this flag is also causing checkInputs like pytest to be no longer provided, which breaks the test right now.

Comment (Mic92) [reply default.nix:20]: Also we still define a checkPhase in default.nix. I think this one should be executed still.

Comment (SuperSandro2000) [reply default.nix:20]: Then we should remove it because dontCheck does not exist.

Comment (Mic92) [reply default.nix:20]: ok.

Comment (SuperSandro2000) [reply default.nix]: force pushed

**Pull request merged**